src/vendor/vendor.c

```diff
@@ -2002,24 +1907,25 @@ void Add_ParamToDM(rdk_component_t *rdkc, char *instantiated_path, char *schema_
     }
 
     // Exit if unable to get the parameter in order to determine its type (and also check that it is readable)
-    ccsp_err = CcspBaseIf_getParameterValues(bus_handle, rdkc->component_name, rdkc->dbus_path, &instantiated_path, 1, &num_values, &values);
-    if (ccsp_err != CCSP_SUCCESS)
+     rbus_err =rbus_getExt(bus_handle, 1,(const char**)&instantiated_path, &num_values, &values);
+
```

Comment:
can you print num_values and values here while performing operations.Introduce a debug